pandas/_libs/properties.pyx

```diff
@@ -12,10 +12,10 @@ cdef class CachedProperty:
     cdef readonly:
         object func, name, __doc__
 
-    def __init__(self, func):
-        self.func = func
-        self.name = func.__name__
-        self.__doc__ = getattr(func, '__doc__', None)
+    def __init__(self, fget):
```

Comment:
`func` -> `fget` to be more `property`-like

Reply:
So this helps catch typing errors of `cache_readonly` since `cache_readonly = property`? I'll mark as resolved.